.semaphore/buckets.yml

```diff
@@ -0,0 +1,26 @@
+version: v1.0
+name: Deploy Buckets
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+
+blocks:
+  - name: Deploy Buckets to all Regions
+    task:
+      prologue:
+        commands:
+          - sem-version python 3.7
+          - checkout
+          - cache restore dev-requirements-$SEMAPHORE_GIT_BRANCH-$(checksum backend/requirements-dev.txt),requirements
+          - cp /home/semaphore/.aws/prime_credentials /home/semaphore/.aws/credentials
+          - source ./project.sh
+          - cz_use_profile thebutton-dev
```

Comment:
can we (I) help here? I want to create permissions that are a little more locked down for this task vs. using a profile from 3 years ago